beacon-chain/blockchain/forkchoice/benchmark_test.go

```diff
@@ -16,7 +16,7 @@ func BenchmarkForkChoiceTree1(b *testing.B) {
 	db := testDB.SetupDB(b)
 	defer testDB.TeardownDB(b, db)
 
-	store := NewForkChoiceService(ctx, db)
+	store := NewForkChoiceService(ctx, db, false)
```

Comment:
can you add a comment on what false is? 
```
store := NewForkChoiceService(ctx, db, false /* ... */)
```

Reply:
yeah, what is false? helps for readability to add that inline comment. pls